.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
+## Reasoning behind the pull request
```

Comment:
please also change L6 to
```
## Proposed changes
```
as seen in rc/v1.4.0 branch

Reply:
done